src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConfigurationClients/FailOverSupportedConfigurationClient.cs

```diff
@@ -0,0 +1,186 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Azure;
+using Azure.Data.AppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.Constants;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.ConfigurationClients
+{
+    internal class FailOverSupportedConfigurationClient : IConfigurationClient
+    {
+        private const int HttpStatusRequestThrottled = 429;
```

Comment:
Is this status code defined in some .net library already?

Reply:
HttpStatusCode.TooManyRequests

[https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=net-6.0#system-net-httpstatuscode-toomanyrequests](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=net-6.0#system-net-httpstatuscode-toomanyrequests)